### PR TITLE
Add support for multiple fileIds in Kyc document

### PR DESCRIPTION
**Pull request by subenson**

### CODE REVIEW

File: openapi/components/schemas/Common/CommonKycDocument.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 type: object
 required:
-  - fileId
+  - fileIds
```
Comment (adamaltman) [inline openapi/components/schemas/Common/CommonKycDocument.yaml:3]: Requiring it is a breaking change.

File: openapi/components/schemas/Common/CommonKycDocument.yaml [hunk 2]
```diff
@@ -13,16 +13,25 @@ properties:
...
     enum:
       - identity-proof
       - address-proof
+  documentSubtype:
+     description: Document subtype submitted for validation.
+     type: string
+     enum:
+      - passport
+      - id-card
+      - drivers-license
```
Comment (adamaltman) [inline openapi/components/schemas/Common/CommonKycDocument.yaml:34]:
```suggestion
      - driver-license
```

File: openapi/components/schemas/Common/CommonKycDocument.yaml [hunk 3]
```diff
@@ -13,16 +13,25 @@ properties:
     description: The сustomer's ID
     allOf:
       - $ref: ../ResourceId.yaml
-  fileId:
-    description: Linked file object id
```
Comment (adamaltman) [inline openapi/components/schemas/Common/CommonKycDocument.yaml:17]: I don't think we can remove it. We can mark it as deprecated. We need to be able to support a transition window...

File: openapi/components/schemas/Common/CommonKycDocument.yaml (hunk 1 above)
Comment (svensono) [reply openapi/components/schemas/Common/CommonKycDocument.yaml]: You're right. I forgot to take BC into account in this repo. This is now  added.

**Review by agualis — APPROVED** [review]

**Review by ZengJunyong — APPROVED** [review]

**Pull request merged**